### PR TITLE
Add shortcuts to context menu

### PR DESCRIPTION
**Pull request by bartsch-dev**
Fixes: https://github.com/koppor/jabref/issues/171.

Added shortcuts to the context menu (just displayed them, didn't create new ones) and refactored the context menu a little bit.

### CODE REVIEW

**Comment by lenhard** [comment]
I tested it manually and LGTM! I really like this feature :-) This makes JabRef look much more like an IDE.

I only hope that all the shortcuts actually work ;-)

File: src/main/java/net/sf/jabref/gui/menus/RightClickMenu.java [hunk 1]
```diff
@@ -67,50 +66,46 @@ public RightClickMenu(JabRefFrame frame, BasePanel panel) {
...
-                if (multiple) {
-                    this.setEnabled(false);
-                }
-            }
-        });
+        add(new GeneralAction(Actions.PASTE, Localization.lang("Paste"), IconTheme.JabRefIcon.PASTE.getSmallIcon(), KeyBinding.PASTE));
+        add(new GeneralAction(Actions.CUT, Localization.lang("Cut"), IconTheme.JabRefIcon.CUT.getSmallIcon(), KeyBinding.CUT));
+        add(new GeneralAction(Actions.DELETE, Localization.lang("Delete"), IconTheme.JabRefIcon.DELETE_ENTRY.getSmallIcon(), KeyBinding.DELETE_ENTRY));
+        GeneralAction printPreviewAction = new GeneralAction(Actions.PRINT_PREVIEW, Localization.lang("Print entry preview"), IconTheme.JabRefIcon.PRINTED.getSmallIcon());
+        printPreviewAction.setEnabled(multiple);
```
Comment (koppor) [inline src/main/java/net/sf/jabref/gui/menus/RightClickMenu.java]: Shouldn't it be `!multiple`?

**Comment by koppor** [comment]
Besides the micro comment: LGTM!

File: src/main/java/net/sf/jabref/gui/menus/RightClickMenu.java (hunk 1 above)
Comment (bartsch-dev) [inline src/main/java/net/sf/jabref/gui/menus/RightClickMenu.java]: Nice catch, fixed.

**Pull request merged**